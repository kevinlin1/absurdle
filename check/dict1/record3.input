renew
savvy
sooth
churn
goofy
stats
lucid
plumb
plump
