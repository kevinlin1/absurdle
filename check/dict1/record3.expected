2309 words
> renew
: ⬜⬜⬜⬜⬜

545 words
> savvy
: ⬜⬜⬜⬜⬜

111 words
> sooth
: ⬜⬜⬜⬜⬜

23 words
> churn
: ⬜⬜🟩⬜⬜

5 words
> goofy
: ⬜⬜⬜⬜⬜

2 words
> stats
: ⬜⬜⬜⬜⬜

2 words
> lucid
: 🟨🟨⬜⬜⬜

2 words
> plumb
: 🟩🟩🟩🟩⬜

1 words
> plump
: 🟩🟩🟩🟩🟩

Absurdle 9/∞

⬜⬜⬜⬜⬜
⬜⬜⬜⬜⬜
⬜⬜⬜⬜⬜
⬜⬜🟩⬜⬜
⬜⬜⬜⬜⬜
⬜⬜⬜⬜⬜
🟨🟨⬜⬜⬜
🟩🟩🟩🟩⬜
🟩🟩🟩🟩🟩
